Dirble: update license and version numbering

sciguy16 opened the pull request:
I've updated the pkgbuild for dirble to match its versioning scheme and license

Edu4rdSHL commented:
I don't understand why we're copying files when we can juts run `cargo build --release` and then use `install -Dm 755 target/release/$pkgname "$pkgdir/usr/bin/$pkgname"`

Any specific reason?

Also, if you want to use a minor `pkgver` number, you need to add `epoch`, see https://wiki.archlinux.org/index.php/PKGBUILD#epoch

Edu4rdSHL commented on packages/dirble/PKGBUILD:
```diff
@@ -2,21 +2,21 @@
 
 pkgname=dirble
-pkgver=186.c1b7237
+pkgver=1.4.0
```
Add epoch.

Edu4rdSHL commented on packages/dirble/PKGBUILD:
```diff
@@ -2,21 +2,21 @@
 source=('git+https://github.com/nccgroup/dirble.git')
 sha512sums=('SKIP')
 
 pkgver() {
```
Remove the whole pkgver() function if you're going to use versioned model.

Edu4rdSHL commented on packages/dirble/PKGBUILD:
```diff
@@ -2,21 +2,21 @@
 url='https://github.com/nccgroup/dirble'
-license=('MIT')
+license=('GPL3')
 makedepends=('git' 'cargo' 'rust')
```
Remove git if you're going to use versioned model.

Edu4rdSHL commented on packages/dirble/PKGBUILD:
```diff
@@ -2,21 +2,21 @@
-license=('MIT')
+license=('GPL3')
 makedepends=('git' 'cargo' 'rust')
 source=('git+https://github.com/nccgroup/dirble.git')
```
Change the source pointing to the release version (like `$pkgname::https://github.com/nccgroup/dirble/archive/v$pkgver.tar.gz`)

ikstream commented:
I think it was on purpose, that we use with git `pkgver` instead of releases

Edu4rdSHL commented on packages/dirble/PKGBUILD:
```diff
@@ -2,21 +2,21 @@
+  awk -F'"' '/^version/ {print $2}' Cargo.toml
 }
 
 build() {
```
Use `cargo build --release` instead of make.

sciguy16 commented:
Thanks for the pointers, I've updated it to fully commit to versioned releases. It only needs cargo as a make dependency, because cargo implies rust. And namcap pointed out that libcurl is a runtime dependency, so I've added that too.
